decapodes.it

```diff
@@ -0,0 +1,137 @@
+@sum Parameter begin
+  LiteralParameter(type::String, value::Float64)
+  ConstantParameter(type::String, value::String)
+end
+
+struct Condition
+  type::String
+  value::String
+  domain_mesh::String
+end
+
+struct DatasetFile
+	uri::String
+	format::String
```

Comment:
Is this string supposed to be a MIME type or from a fixed list? If it is from a list, that should be a sum type too.